src/main/java/com/olgaboiar/mint/Response.java

```diff
@@ -20,7 +22,7 @@ Body getBody() {
 
     String prepareResponse() {
         if (body.getBodyString().length() > 0) {
-            responseToSend = header.prepareHeaders() + "\n" + body.getBodyString();
+            responseToSend = header.prepareHeaders() + BLANK_LINE + body.getBodyString();
```

Comment:
Test coverage!

Reply:
added in https://github.com/olgaboiar/MintHTTPServer/pull/10/commits/f5d66a5a076d08a5ba4974bff8fff68aa4386a99